src/reformat.c

```diff
@@ -138,6 +414,13 @@ avifResult avifImageRGBToYUV(avifImage * image, const avifRGBImage * rgb)
         avifImageAllocatePlanes(image, AVIF_PLANES_A);
     }
 
+    if (state.mode == AVIF_REFORMAT_MODE_IDENTITY && image->depth == 8 && image->yuvFormat == AVIF_PIXEL_FORMAT_YUV444 && image->yuvRange == AVIF_RANGE_FULL) {
```

Comment:
Minor style change: Can you please parenthesize the inner conditionals the way the other checks do in this function?